GUIDELINES_AND_INFO.md

```diff
@@ -0,0 +1,20 @@
+# ScratchVerifier Information
+## What this is and is not
+This is not:
+- a platform for Scratchers to use except as an API for programs
+- a website that allows you to link accounts, or a social media
+
+This is
+- an API for Scratch account verification
+
+## Guidelines
+(NOT DONE)
+Please read our [Code of Conduct](https://github.com/ScratchVerifier/ScratchVerifier/blob/develop/CODE_OF_CONDUCT.md) before proceeding. 
+The following behaviour is not encouraged:
+- Posting issues with very simple descriptions, or unconstructive criticism. We would like you to in detail explain what can be done, what is wrong and what could get an update.
```

Comment:
Say "explain in detail" instead of "in detail explain".

Reply:
Also, maybe mention how issue templates are a good start for what information should be included in an issue.

Reply:
done